tests/test_index_api.py

```diff
@@ -28,7 +28,7 @@ def test_get_index(client, index_name, create_index, fmt):
         json={
             "changes": [
                 {"insert": {"id": 1, "hashes": [101, 201, 301]}},
-                {"set_attribute": {"name": "foo", "value": 1234}},
+                {"set_metadata": {"name": "foo", "value": "1234"}},
```

Comment:
This operation should no longer be allowed in the API. It should use the new metadata field.